src/test/controllers/users.controller.test.js

```diff
@@ -222,6 +221,57 @@ describe('Test User API', function () {
   });
 });
 
+describe('/job', function () {
+  it('request /job without session coockie.', done => {
+    anauthorizedAccessTest(API_BASE_URL + '/job', done);
+  });
+
+  it('request /job with session coockie.', done => {
+    rp({
+      method: 'GET',
+      uri: API_BASE_URL + '/job',
+      jar: true,
+      resolveWithFullResponse: true,
+      json: true,
+    })
+      .then(function (result) {
+        result.statusCode.should.equal(200);
+        let body = result.body;
+        body.area.toString().should.equal(jobData.area.toString());
+        body.years.toString().should.equal(jobData.years.toString());
+        body.education_background.toString()
+          .should.equal(jobData.education_background.toString());
+        done();
+      })
+      .catch(function (err) {
+        should.fail();
+        done();
+      });
+  });
+});
+
+describe('/editOption', function () {
+  it('request /signup with session coockie.', done => {
+    rp({
+      method: 'POST',
+      uri: API_BASE_URL + '/editOption',
```

Comment:
So put that template in String?
`uri: '${API_BASE_URL}/editOption',`
like this?